tasks/src/main/scala/dagr/tasks/picard/CollectQualityYieldMetrics.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2017 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+package dagr.tasks.picard
+
+import java.nio.file.Path
+
+import dagr.tasks.DagrDef.{PathToBam, PathToFasta}
+
+import scala.collection.mutable.ListBuffer
+
+
+object CollectQualityYieldMetrics {
+  def metricsExtension: String = ".quality_yield_metrics"
+}
+
+class CollectQualityYieldMetrics(override val in: PathToBam,
+                                 override val prefix: Option[Path] = None,
+                                 secondary: Option[Boolean] = None,
```

Comment:
Could these by `includeSecondary` or `incSecondary`?  `secondary` alone made me go look at how it was being used because I wasn't 100% sure what it meant.